robot/EDA/resources/EDA_robot_backup.robot

```diff
@@ -0,0 +1,217 @@
+*** Settings ***
```

Comment:
is the intention of this file to hold unused api keywords that may be useful for future?

Reply:
Yup. I added the intention to documentation of this file. So that if someone else takes over EDA repo, they know why the backup file exists. I know it looks like a tech debt, but I feel once new WIs are picked up related to courses and program plans, these keywords will come to good use.